PP_APP/server/pp_api/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.conf import settings
+
+# Create your models here.
+
+class hello_rest(models.Model):
+    name = models.CharField(max_length=20, null=False, blank=False)
+    msg = models.CharField(max_length=250, null=False, blank=False)
+
```

Comment:
i think we should comment out the unnecessary lines if it is pushing to the master. I know you might want to leave it for reference, so maybe comment out + leave a comment, is good enough